core/trino-main/src/main/java/io/trino/testing/LocalQueryRunner.java

```diff
@@ -474,7 +475,7 @@ private LocalQueryRunner(
                 new AnalyzePropertiesSystemTable(metadata, accessControl, analyzePropertyManager),
                 new TransactionsSystemTable(typeManager, transactionManager)),
                 ImmutableSet.of(),
-                ImmutableSet.of(new ExcludeColumnsFunction()));
+                ImmutableSet.of(new ExcludeColumnsFunction(), new SequenceFunction()));
```

Comment:
Not needed?

Reply:
needed for the regression test query. will remove it, because i remove the regression test query
thanks